README.md

```diff
@@ -369,8 +369,8 @@ Other Style Guides
     const itemsCopy = [...items];
     ```
 
-  <a name="arrays--from"></a><a name="4.4"></a>
-  - [4.4](#arrays--from) To convert an array-like object to an array, use spreads `...` instead of [`Array.from`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from).
+  <a name="arrays--from-iterable"></a><a name="4.4"></a>
```

Comment:
we'll need to keep an `<a name="arrays--from"></a>` so that existing links don't break.